### PR TITLE
Replication

### PR DESCRIPTION
**Pull request by FurkanS1821**
- Replication
- `CooldownsEnabled` and `ManaCostsEnabled` variables are now used
- Updated travis and appveyor scripts
- Commands now increase the stat, not directy set it.
- Changed settings of `LeagueSandbox-Default` project, it won't compile the scripts for no reason anymore (im not sure this option was in the .csproj file, it might not be pushed because of `.gitignore`)

### CODE REVIEW

**Comment by HarmfulBreeze** [comment]
You should also mention that this PR modifies various commands such as the AD command that adds AD and doesn't set to a specific amount anymore

**Comment by FurkanS1821** [comment]
@piorrro33 I believe that should have always been the behaviour since that would cause things to go wrong when mixed with temporary buffs. (i.e. a buff adds 50 ad, you reset that via command, then buff removes 50 ad, you get base-50 ad)

File: GameServerLib/Logic/GameObjects/Stats/Stats.cs [hunk 1]
```diff
@@ -225,50 +233,103 @@ public float GetLevel()
...
+            {
+                SpellsEnabled &= ~(1u << id);
+            }
         }
 
-        public float getCritDamagePct()
+        public bool GetSummonerSpellEnabled(byte id)
         {
-            return critDamagePct;
+            return (SummonerSpellsEnabled & 1u << id) != 0;
```
Comment (moonshadow565) [inline GameServerLib/Logic/GameObjects/Stats/Stats.cs]: SetSummonerSpellEnable uses ``16u``, maybe you should use ``16u`` here too ?

**Comment by HarmfulBreeze** [comment]
I agree, but that's a change so I believe you should mention it

**Review by moonshadow565 — APPROVED** [review]
Everything should work.
At least it doesn't crash :)
Don't delete the branch in case we need to fix something.

**Pull request merged**